src/jumpstart/JumpstartClaimStatusToasts.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import JumpstartClaimStatusError from 'src/jumpstart/JumpstartClaimStatusError'
+import JumpstartClaimStatusLoading from 'src/jumpstart/JumpstartClaimStatusLoading'
+
+export default function JumpstartClaimStatusToasts() {
```

Comment:
nit: sometimes i feel like reading react components is like peeling an onion which can be annoying when there are too many layers...i wonder if we need this layer in the middle? the loading and error toast components are relatively short, perhaps we can have them together in one component?

Reply:
we certainly don't need them to be separated. I was thinking about encapsulating all logic specific to a single toast to its own component. but yeah looks like it's overkill in this case.